requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3==1.17.105
```

Comment:
This file is for the publish script right? Maybe name it publish-requirements.txt for more clarity?